### PR TITLE
Rephrase the bit about the UA chosing the push service

### PR DESCRIPTION
**Pull request by beverloo**
***
[Preview](https://s3.amazonaws.com/pr-preview/beverloo/push-api/privacy-remark.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/push-api/c35f015...beverloo:9cbd303.html)

### CODE REVIEW

**Comment by beverloo** [comment]
/cc @chaals 

This is more explicit than it used to be, while removing the assertion that there's only one. (Addressing Firefox' configuration option.) Does this address your concerns sufficiently, Charles?

File: index.html [hunk 1]
```diff
@@ -486,11 +486,17 @@
...
+          creating <a>push subscriptions</a>. There usually is a pre-existing relationship between
+          the <a>user agent</a> and the <a>push service</a> due to a variety of performance-related
+          concerns, including the complexity of running reliable <a>push services</a> and the impact
+          on battery lifetime if there were an unbounded set of <a>push services</a> to which a
+          device could connect.
+        </p>
+        <p class="note">
+          While the contents of a <a>push message</a> are encrypted [[!WEBPUSH-ENCRYPTION]], the
+          <a>push service</a> is still exposed to the metadata of messages sent to a
+          <a>push subscription</a> such as their frequency, timing and size.
```
Comment (martinthomson) [inline index.html]: Add source and destination.  Maybe add: "Only size can be mitigated, by the addition of padding."

**Review by martinthomson — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (beverloo) [reply index.html]: Done.

**Comment by chaals** [comment]
Not really. It fails to mention the issue that a given push service may also be blocked by a firewall, and it still implies there can be only one.

I would suggest phrasing it in such a way as to clarify that user agents may restrict the number of push services, and may only offer those with which there is a pre-existing relationship, and then clarify the pros and cons (privacy/security, reliability, battery consumption).

**Comment by chaals** [comment]
ping?

**Comment by LJWatson** [comment]
@Beverloo @martinthomson can one of you respond to @Chaals' comments, so we can get this PR merged and the spec on its way? Thanks.

**Comment by LJWatson** [comment]
@Chaals if you feel strongly about this, please propose some concrete text that builds on your comments of [Aug 22](https://github.com/w3c/push-api/pull/287#issuecomment-324001428). Thanks.

**Comment by martinthomson** [comment]
I'm merging this, and I think that the ball is with @chaals.  Happy to take a PR that clarifies this point, but I just don't have the time to write it myself.

**Comment by chaals** [comment]
Yup. Wrote that PR - #294

**Pull request merged**